cassiopeia/solver/DistanceSolver.py

```diff
@@ -11,11 +11,17 @@
 import networkx as nx
 import numpy as np
 import pandas as pd
+import configparser
+import subprocess
+import os
+import tempfile
+import ete3
 
 from cassiopeia.data import CassiopeiaTree
 from cassiopeia.mixins import DistanceSolverError
-from cassiopeia.solver import CassiopeiaSolver, solver_utilities
-
+from cassiopeia.solver import (CassiopeiaSolver, 
+    solver_utilities)
+from cassiopeia.data.utilities import ete3_to_networkx
```

Comment:
Better to import `from cassiopeia.data import utilities as data_utilities` and then below call `data_utilities.ete3_to_networkx`. Just improves readability for users not familiar with the codebase.